Files/Views/SettingsPages/Appearance.xaml.cs

```diff
@@ -25,7 +25,7 @@ public Appearance()
             ThemeChooser.SelectedIndex = (int)Enum.Parse(typeof(ElementTheme), ThemeHelper.RootTheme.ToString());
             ThemeChooser.Loaded += (s, e) =>
             {
-                ThemeChooser.SelectionChanged += async (s1, e1) =>
```

Comment:
What purpose of this change?

Reply:
Without `await` in the closure, `async` here is unnecessary but adds overhead. VS also produces warnings for it.

Reply:
Did you test this change? Is changing theme working properly?

Reply:
Sorry for the out-of-scope change, but I've tested it and it works fine.
![image](https://user-images.githubusercontent.com/14960345/85141477-698f3000-b279-11ea-9a73-5d65581484f5.png)
![image](https://user-images.githubusercontent.com/14960345/85141502-714ed480-b279-11ea-8d5e-b5345ce541c0.png)